installing/installing_cloud_other/install-rhacs-cloud-other.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: ASSEMBLY
+[id="install-rhacs-cloud-other"]
+= Setting up RHACS Cloud Service on other platforms
+include::modules/common-attributes.adoc[]
+:context: install-rhacs-cloud-other
+:topic-helm:
+
+toc::[]
+
+[role="_abstract"]
+{product-title-managed} ({product-title-managed-short}) provides security services for your Kubernetes clusters. You can install {product-title-managed-short} on Kubernetes systems with secured clusters on managed platforms such as Amazon Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE), and Microsoft Azure Kubernetes Service (AKS).
+
+:FeatureName: {product-title-managed-short}
+include::snippets/field-trial.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* Understand the xref:../../installing/acs-installation-platforms.adoc#install-platforms-methods[installation platforms and methods].
+* Understand xref:../../architecture/acs-architecture.adoc#acs-architecture_acs-architecture[{product-title} architecture].
+* Review the xref:../installing_cloud_other/prerequisites-cloud-other.adoc#prerequisites-cloud-other[prerequisites].
+* Ensure that the Central custom resource has been installed and configured by Red Hat.
+
+To set up {product-title-managed-short}:
+
+. In the {cloud-console}, from the navigation menu, select *Advanced Cluster Security* -> *ACS Instances* and select the instance to which you want to connect your secured clusters. 
+* In the *Instance Details* section, note the *Central API Endpoint*. This address is used when creating secured clusters.
```

Comment:
I am not clear on the second sentence
s/This address is used/You use this address or This is the address that was used ??